packages/plugins/rum/src/built/rum-browser-sdk.ts

```diff
@@ -0,0 +1,17 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+/* global globalThis */
+
+import { datadogRum } from '@datadog/browser-rum';
+
+// To please TypeScript.
+const globalAny: any = globalThis;
+
+// Also them to the global DD_RUM object.
+globalAny.DD_RUM = globalAny.DD_RUM || {};
+globalAny.DD_RUM = {
+    ...globalAny.DD_RUM,
+    ...datadogRum,
```

Comment:
I don't have any knowledge on whether `window.DD_RUM` and `datadogRum` share the exact same API. I'll let @Miz85 review this one

Reply:
From [the documentation website](https://docs.datadoghq.com/real_user_monitoring/browser/setup/client?tab=rum):

<img width="774" alt="image" src="https://github.com/user-attachments/assets/e68fa7f3-a632-472f-b785-48507bf44bc4" />

I'd expect the two to be the same.

Reply:
Yes they are the same.

But we should choose on strategy for the injection and roll with that, why do we want to support both the CDN and the npm install methods?